staff/sergio-luz/js-arrays/every.js

```diff
@@ -0,0 +1,14 @@
+function every(arr, callback){
+    if(!(callback instanceof Function) && !(arr instanceof Array)) throw Error ('arr is not an array and callback is not a function');
+    if( !(arr instanceof Array)) throw Error('arr is not an array');
```

Comment:
Maybe "arr" could be outside of the console.log to give more info to the user (lines 2, 3 and 4)

Reply:
According to every-test,js  it has to be inside. That is the only way to be able to complete the last test. (Remember that we can not change the test of our classmates)